x509util/templates.go

```diff
@@ -107,6 +108,17 @@ func (t TemplateData) SetCertificateRequest(cr *x509.CertificateRequest) {
 	t.SetInsecure(CertificateRequestKey, newCertificateRequest(cr))
 }
 
+// SetWebhooks sets the given webhook response in the webhooks template data.
+func (t TemplateData) SetWebhooks(webhookName string, data interface{}) {
```

Comment:
It looks like it's assigning only one webhook:
```suggestion
// SetWebhook sets the given webhook response in the webhooks template data.
func (t TemplateData) SetWebhook(webhookName string, data interface{}) {
```

Reply:
We also should add a method like this to `sshutil/templates.go`

Reply:
ade8a1f